tools/docker/tests/openwrt/test_status.sh

```diff
@@ -24,3 +24,5 @@ done
 exit 1
 EOF
 
+echo "Stopping prplMesh"
```

Comment:
We need to stop it at the end of the test to avoid the multiple target from interfering with one another.
Now this one is not strictly needed because we will kill all the instances before the tests starts, but it gives it a chance to stop more gracefully.